Source/JavaScriptCore/runtime/IntlNumberFormat.cpp

```diff
@@ -475,6 +475,10 @@ ASCIILiteral IntlNumberFormat::partTypeString(UNumberFormatFields field, double
     case UNUM_EXPONENT_FIELD:
 #if !defined(U_HIDE_DEPRECATED_API)
     case UNUM_FIELD_COUNT:
+#endif
+#if !defined(U_HIDE_DRAFT_API)
```

Comment:
what for is this define?

Reply:
It's from ICU, used in ICU header that we include, if undefined then those 2 additional "draft" enum values are visible